htdocs/modules/system/templates/system_userinfo.tpl

```diff
@@ -167,8 +167,10 @@
         <img src="<{$result.image}>" alt="<{$module.name}>"/>
         <strong><a href="<{$result.link}>" title="<{$result.title}>"><{$result.title}></a></strong>
         <br>
-        <span class="x-small">(<{$result.time}>)</span>
```

Comment:
We can not change only the template, there is a php error:
> Notice: Undefined index: time in file /userinfo.php line 184
I modify the templates. You have to keep the modification of the php file!